eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1385,7 +1386,7 @@ protected function internalUpdateContent(
                 );
                 $fieldValues[$fieldDefinition->identifier][$languageCode] = $fieldValue;
 
-                if ($isRetained || $isCopied || ($isLanguageNew && $isEmpty) || $isProcessed) {
+                if ($isRetained || $isCopied || ($isLanguageNew && $isEmpty && !$copyEmptyField) || $isProcessed) {
```

Comment:
```suggestion
                if ($isRetained || $isCopied || $isLanguageNew || $isProcessed) {
```

`internalUpdateContent` is a method used only it two places and, correct me if I am wrong, the change does not seem to affect `ContentService::contentUpdate` much/at all. The parameter isn't exposed beyond protected scope so it basically is a hardcoded behaviour.

Reply:
@Nattfarinn no, this simply won't make it. You can immediately see that integration tests fail one by one if we apply this change. 

This we have to skip this `if` logic only when we are copying translations from already published version.